drivers/adc/ad7124/ad7124_regs.c

```diff
@@ -1,3 +1,44 @@
+/***************************************************************************//**
+*   @file    AD7124-regs.c
```

Comment:
it's a good idea to keep filename consistency here;
the filename is `ad7124_regs.c`, so it would be a good idea to also name this the same;

Reply:
done